docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -1,6 +1,66 @@
 import { Alert,  Tabs, TabItem } from '@aws-amplify/ui-react';
 import { TerminalCommand } from '@/components/InstallScripts';
 
+## 4.x to 5.x
+### Installation
+Install the 5.x version of the `@aws-amplify/ui-react` library.
+
+<Tabs>
+<TabItem title="npm">
+<TerminalCommand command="npm install @aws-amplify/ui-react@5.x" />
+</TabItem>
+<TabItem title="yarn">
+<TerminalCommand command="yarn add @aws-amplify/ui-react@5.x" />
+</TabItem>
+</Tabs>
+
+### Update and usage
+
+`@aws-amplify/ui-react@5.x` introduces the following breaking changes:
+
+#### 1. `@aws-amplify/ui-react@5.x` drops Amplify Geo components. They are moved to a separate new package `@aws-amplify/ui-react-geo`.
+Install the 1.x version of the `@aws-amplify/ui-react-geo` library and update your existing imports accordingly.
+
+<Tabs>
+<TabItem title="npm">
+<TerminalCommand command="npm install @aws-amplify/ui-react-geo@1.x" />
+</TabItem>
+<TabItem title="yarn">
+<TerminalCommand command="yarn add @aws-amplify/ui-react-geo@1.x" />
+</TabItem>
+</Tabs>
+
+Styles supporting Amplify Geo UI components are moved to `@aws-amplify/ui-react-geo` library and are not available through `@aws-amplify/ui-react` library anymore.
```

Comment:
Can we add an example?

Reply:
Added in https://github.com/aws-amplify/amplify-ui/pull/4007/commits/48000eb5eab6f7fb07df313fd6c7bf9d82423967